src/cpp/session/modules/SessionTutorial.R

```diff
@@ -247,7 +247,18 @@
    
    tutorialFile <- tutorialFiles[[1]]
    contents <- readLines(tutorialFile, encoding = "UTF-8", warn = FALSE)
-   yaml <- rmarkdown:::parse_yaml_front_matter(contents)
+   
+   # find YAML header (if any). note that we avoid using rmarkdown here
+   # just because we want to avoid 
```

Comment:
Incomplete comment?

Reply:
Oops. Looks like I lost my train of thought!